correlation_context/HTTP_HEADER_FORMAT_RATIONALE.md

```diff
@@ -1,74 +1,72 @@
 # Correlation Context Header Format Rationale
 
-This document provides rationale for the decisions made for the `Correlation-Context` header format.
+This document provides a rationale for the decisions made for the `Correlation-Context` header format.
 
 ## General considerations
 
-- Should be human readable. Cryptic header will hide the fact of potential information disclosure.
-- Should be appende-able (comma-separated) https://tools.ietf.org/html/rfc7230#page-24 so nodes 
-can add context properties without parsing an existing headers.
-- It is expected that the typical name will be a single word in latin and the value will be a 
-short string in latin or a derivative of an url.
+- It should be human-readable. Cryptic headers would hide the fact of potential information disclosure.
+- It should be append-able (comma-separated) https://tools.ietf.org/html/rfc7230#page-24 so nodes
+can add context properties without parsing existing headers.
+- Keys are a single word in ASCII, and values should be a short string in UTF-8 or a derivative of an URL.
 
 ## Why a single header?
 
-Another option is to use prefixed headers, such as `trace-context-X` where `X` is a propagated 
-field name. This could reduce the size of the data, particularly in http/2 where header 
+Another option would be to use prefixed headers, such as `trace-context-X`, where `X` is a propagated
+field name. That could reduce the size of the data, particularly in http/2, where header
 compression can apply.
 
-Generally speaking `Correlation-Context` header may be split into multiple headers and 
-compression may be at the same ballpark as repeating values will be converted into a single value
-in HPAC's dynamic collection. That's said there were no profiling made to make this decision.
+Generally speaking, a `Correlation-Context` header may be split into multiple headers, and
+compression may be at the same ballpark as repeating values are converted into a single value
+in HPAC's dynamic collection. That said, there was no profiling made to make this decision.
 
 The approach with multiple headers has the following problems:
-- Name values limitation is much more pressing when the context name is used a part of a header 
+- Name values limitation is much more pressing when the context name is used a part of a header
```

Comment:
Not sure what "Name values" means. We should rephrase.

Reply:
https://github.com/w3c/correlation-context/issues/26